modules/nodes-pods-secrets-creating-web-console-secrets.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-secrets-creating-web-console-secrets_{context}"]
+= Creating a secret using the web console
```

Comment:
There are three demands in the [OSDOCS-3301](https://issues.redhat.com/browse/OSDOCS-3301), but I think that this file only meet "As a user, I want to add secrets from the web console". These two need to be added:
- As a user, I want to manage secrets from the web console. 
- As a user, I want to add secrets as environment variables (via the web console and CLI), such as https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Reply:
Thank you bringing this up. The second demand about adding [secrets as environment variables](https://docs.openshift.com/container-platform/4.13/nodes/pods/nodes-pods-secrets.html#nodes-pods-secrets-creating_nodes-pods-secrets) is already addressed in example with the title "YAML of a pod populating environment variables with secret data", so that task will be removed from the to-do. I will work on the managing secrets portion.